backend/backend/src/main/java/de/aaaaaaah/velcom/backend/restapi/endpoints/QueueEndpoint.java

```diff
@@ -198,6 +198,34 @@ public PostCommitReply addCommit(
 		return new PostCommitReply(JsonTask.fromTask(insertedTask.get(), commitReadAccess));
 	}
 
+
+	@POST
+	@Path("commit/{repoId}/{hash}/one-up")
```

Comment:
I just noticed that the queue endpoint seems to use `repoId` instead of `repoid` like the API spec and other endpoints do. Also, the spec task endpoint uses `taskId` instead of `taskid`. Slightly inconsistent overall. Maybe we could fix it in this PR?